connector/amqp10/amqp10.go

```diff
@@ -89,9 +113,6 @@ func ConnectAMQP10(cfg config.Config, logger *logging.Logger) (*AMQP10Connector,
 	if err != nil {
 		return nil, err
 	}
-	if opt == nil {
-		return nil, fmt.Errorf("Failed to get connection URL from configuration file")
-	}
```

Comment:
I see a bunch of changes around getting these options from the config. I see, that everywhere else, there is a `if opt != nil && err == nil` line, after which you either use a value from a config file, or use a default value. I don't see that here. What happens if opt is nil here?

Reply:
Ah good catch. That is actually a bug. Program would fail later in the process, but with less useful output. Will put it back.